LayoutTests/TestExpectations

```diff
@@ -5319,6 +5319,7 @@ fast/attachment/attachment-image-controls-basic.html [ Skip ]
 # This test requires a system font that has Bulgarian-specific shaping.
 fast/text/bulgarian-system-language-shaping.html [ ImageOnlyFailure ]
 
-# Temporarily disable accessibility bold tests while it's being reimplemented.
-webkit.org/b/237817 fast/text/accessibility-bold-system-font.html [ Pass Failure ]
-webkit.org/b/237817 fast/text/accessibility-bold-system-font-2.html [ Pass ImageOnlyFailure ]
+# Accessibility bold only exists on iOS.
+fast/text/accessibility-bold-system-font.html [ Pass Failure ]
```

Comment:
`Pass Failure` is an odd way to express this. Just skip on all platforms that aren't iOS?

Reply:
I thought `Pass Failure` was recommended, to make sure the test doesn't crash on other OSes?